java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -170,6 +172,23 @@ public void testCallingPythonActor() {
     Assert.assertEquals(res.get(), "2".getBytes());
   }
 
+  @Test
+  public void testCallingCppFunction() {
+    ObjectRef<Integer> res = Ray.task(CppFunction.of("Plus", Integer.class), 1, 2).remote();
```

Comment:
Can you add some comments? E.g., the first case is "Test calling a simple C++ function.", the second is "Test calling a C++ function that returns a large object."